app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryViewModel.kt

```diff
@@ -77,6 +77,14 @@ class StorySummaryViewModel(
           ephemeralChapterSummary.chapterSummary.title,
           ephemeralChapterSummary.writtenTranslationContext
         ),
+        previousChapterTitle = if (index > 0) {
+          translationController.extractString(
+            ephemeralStorySummary.chaptersList[index - 1].chapterSummary.title,
+            ephemeralStorySummary.chaptersList[index - 1].writtenTranslationContext
+          )
+        } else {
+          ""
+        },
```

Comment:
Perhaps instead pass in null and add the null check to the 'when' condition for MISSING_PREREQUISITES? I think that will lead to a slightly cleaner & more robust implementation.

Reply:
I have added the else statement as suggested. It returns  a generic string that I also added to address the usecase where there is no previous chapter title.